<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_path">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>681</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/andreas/kdtools/test-data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_query">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>(?:weak|strict) correctness assumption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_regex">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RegEx</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_case_sensitive">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>case-sensitive</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_file_format_pdf">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>.pdf</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_file_format_docx">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.doc(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_file_format_txt">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.txt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_file_format_html">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>html</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_file_format_xlsx">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.xls(x)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_file_format_pptx">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.ppt(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_config">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save config</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load_config">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load config</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuThemes">
    <property name="title">
     <string>Themes</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuThemes"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_directory">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDocs">
   <property name="text">
    <string>Docs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
